easybuild/easyconfigs/p/plantri/plantri-5.4-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'MakeCp'
+
+name = 'plantri'
+version = '5.4'
+
+homepage = 'https://doc.sagemath.org/html/en/reference/spkg/plantri.html'
+description = """Plantri is a program that generates certain types of graphs that are imbedded on the sphere.
```

Comment:
@PetrKralCZ if you're feeling enthusiastic you could report the misspelling of "embedded" here to the upstream developers.